src/loki/values/upstream-values.yaml

```diff
@@ -22,4 +22,4 @@ memcached:
 sidecar:
   image:
     repository: ghcr.io/kiwigrid/k8s-sidecar
-    tag: 1.30.10
+    tag: 2.1.4
```

Comment:
This update should be avoided for now (see https://github.com/defenseunicorns/uds-core/pull/2207).